<launch>
  <rosparam
    ns="depth_pose_estimator"
    command="load"
    file="$(find depth_pose_estimator)/params/double_depth_pose_estimator_params.yaml"
  />

  <node
    name="left_depth_pose_estimator"
    pkg="depth_pose_estimator"
    type="double_depth_pose_estimator"
    output="screen"
  />

  <node
    name="right_depth_pose_estimator"
    pkg="depth_pose_estimator"
    type="double_depth_pose_estimator"
    output="screen"
  />
</launch>